### PR TITLE
Update regions-ip-addresses.md

### PR DESCRIPTION
**Pull request by mirnawong1**
adding multi tenant to regions in order to clarify confusion amongst users. all of our documents specify multi tenant (e.g. this features is available on multi tenant enterprise plans) and may link to this page, however this page makes no reference to multi tenant. users also don't understand the north American region doesn't mean they have to be physically located in north america.

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/deploy/regions-ip-addresses.md [hunk 1]
```diff
@@ -4,16 +4,16 @@ id: "regions-ip-addresses"
...
 
 [dbt Cloud Enterprise](https://www.getdbt.com/pricing/) plans can choose to have their account hosted in any of the below regions. Organizations **must** choose a single region per dbt Cloud account. If you need to run dbt Cloud in multiple regions, we recommend using multiple dbt Cloud accounts. 
 
 
 | Region | Location | Access URL | IP addresses | Developer plan | Team plan | Enterprise plan |
 |--------|----------|------------|--------------|-----------------|------------|------------------|
-| North America | us-east-1 (N. Virginia) | cloud.getdbt.com | 52.45.144.63 <br /> 54.81.134.249 <br />52.22.161.231 | ✅ | ✅ | ✅ |
-| EMEA | eu-central-1	(Frankfurt) | emea.dbt.com | 3.123.45.39 <br /> 3.126.140.248 <br /> 3.72.153.148 | ❌ | ❌ | ✅ |
-| APAC  | 	ap-southeast-2  (Sydney)| au.dbt.com | 52.65.89.235 <br /> 3.106.40.33 <br /> 13.239.155.206 <br />| ❌ | ❌ | ✅ |
+| North America <br />multi tenant | us-east-1 <br />(N. Virginia) | cloud.getdbt.com | 52.45.144.63 <br /> 54.81.134.249 <br />52.22.161.231 | ✅ | ✅ | ✅ |
```
Comment (runleonarun) [inline website/docs/docs/deploy/regions-ip-addresses.md]:
```suggestion
| North America[^*] | us-east-1 <br />(N. Virginia) | cloud.getdbt.com | 52.45.144.63 <br /> 54.81.134.249 <br />52.22.161.231 | ✅ | ✅ | ✅ |
```

File: website/docs/docs/deploy/regions-ip-addresses.md [hunk 2]
```diff
@@ -4,16 +4,16 @@ id: "regions-ip-addresses"
...
+| EMEA <br />multi tenant | eu-central-1	(Frankfurt) | emea.dbt.com | 3.123.45.39 <br /> 3.126.140.248 <br /> 3.72.153.148 | ❌ | ❌ | ✅ |
```
Comment (runleonarun) [inline website/docs/docs/deploy/regions-ip-addresses.md]:
```suggestion
| EMEA[^*] | eu-central-1	(Frankfurt) | emea.dbt.com | 3.123.45.39 <br /> 3.126.140.248 <br /> 3.72.153.148 | ❌ | ❌ | ✅ |
```

File: website/docs/docs/deploy/regions-ip-addresses.md [hunk 3]
```diff
@@ -4,16 +4,16 @@ id: "regions-ip-addresses"
...
+| APAC  <br />multi tenant | 	ap-southeast-2  (Sydney)| au.dbt.com | 52.65.89.235 <br /> 3.106.40.33 <br /> 13.239.155.206 <br />| ❌ | ❌ | ✅ |
```
Comment (runleonarun) [inline website/docs/docs/deploy/regions-ip-addresses.md]:
```suggestion
| APAC [^*] | 	ap-southeast-2  (Sydney)| au.dbt.com | 52.65.89.235 <br /> 3.106.40.33 <br /> 13.239.155.206 <br />| ❌ | ❌ | ✅ |
```

Comment (runleonarun) [inline website/docs/docs/deploy/regions-ip-addresses.md]:
```suggestion
| APAC[^*] | 	ap-southeast-2  (Sydney)| au.dbt.com | 52.65.89.235 <br /> 3.106.40.33 <br /> 13.239.155.206 <br />| ❌ | ❌ | ✅ |
```

File: website/docs/docs/deploy/regions-ip-addresses.md [hunk 4]
```diff
@@ -4,16 +4,16 @@ id: "regions-ip-addresses"
...
 | Virtual Private dbt or Single tenant | Customized |  Customized | Ask [Support](/guides/legacy/getting-help#dbt-cloud-support) for your IPs | ❌ | ❌ | ✅ |
```
Comment (runleonarun) [inline website/docs/docs/deploy/regions-ip-addresses.md:18]: [^*]: These regions support multi-tenant instances only.

**Review by runleonarun — COMMENTED** [review]
@mirnawong1 What do you think if we did something like my suggestions? We might need to use a number instead of an asterisk. We can test how it looks if we commit my code suggestions.

File: website/docs/docs/deploy/regions-ip-addresses.md (hunk 4 above)
Comment (mirnawong1) [reply website/docs/docs/deploy/regions-ip-addresses.md:18]: i've made a small tweak to this @runleonarun. According to my search, it doesn't look like the footnote support asterisks, only footnote numbers.

**Comment by mirnawong1** [comment]
i've made a small tweak to this @runleonarun. According to my search, it doesn't look like the footnote support asterisks, only footnote numbers.

**Comment by mirnawong1** [comment]
hey @runleonarun can you review this again? thank yoU!

**Review by runleonarun — APPROVED** [review]
This looks great Mirna!

**Pull request merged**